src/Gitee/composer.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "socialiteproviders/gitee",
+    "description": "Gitee OAuth2 Provider for Laravel Socialite",
+    "license": "MIT",
+    "authors": [{
+        "name": "mowangjuanzi",
+        "email": "baoguoxiao0538@hotmail.com"
+    }],
+    "require": {
+        "php": "^5.6 || ^7.0",
```

Comment:
can drop < 7.0

Reply:
ok. Let me check it out first

Reply:
https://github.com/SocialiteProviders/Documentation/pull/104